src/lib.rs

```diff
@@ -29,6 +29,7 @@ use serde::de::DeserializeOwned;
 use tokio::time::sleep;
 
 mod types;
+use crate::types::LOAD_EXISTING_ACCOUNT;
```

Comment:
I think this import should be moved into the existing `use types::{ ... }` group on L40..43